test/Common/TestActivities.cs

```diff
@@ -12,50 +14,73 @@ internal static class TestActivities
     {
         public const char BigValueChar = '*';
 
-        public static string Hello([ActivityTrigger] DurableActivityContextBase ctx)
+        public static string Hello([ActivityTrigger] IDurableActivityContext ctx)
         {
             string input = ctx.GetInput<string>();
             return $"Hello, {input}!";
         }
 
-        public static object Echo([ActivityTrigger] DurableActivityContext ctx)
+        public static object Echo([ActivityTrigger] IDurableActivityContext ctx)
         {
             object obj = ctx.GetInput<object>();
             return obj;
         }
 
-        public static long Multiply([ActivityTrigger] DurableActivityContext ctx)
+        public static long Multiply([ActivityTrigger] IDurableActivityContext ctx)
         {
             (long a, long b) = ctx.GetInput<(long, long)>();
             return a * b;
         }
 
-        public static long Add([ActivityTrigger] DurableActivityContext ctx)
+        public static long Add([ActivityTrigger] IDurableActivityContext ctx)
         {
             (long a, long b) = ctx.GetInput<(long, long)>();
             return a + b;
         }
 
-        public static string[] GetFileList([ActivityTrigger] DurableActivityContext ctx)
+        public static string[] GetFileList([ActivityTrigger] IDurableActivityContext ctx)
         {
             string directory = ctx.GetInput<string>();
             string[] files = Directory.GetFiles(directory, "*", SearchOption.TopDirectoryOnly);
             return files;
         }
 
-        public static long GetFileSize([ActivityTrigger] DurableActivityContext ctx)
+        public static long GetFileSize([ActivityTrigger] IDurableActivityContext ctx)
         {
             string fileName = ctx.GetInput<string>();
             var info = new FileInfo(fileName);
             return info.Length;
         }
 
-        public static void ThrowActivity([ActivityTrigger] DurableActivityContext ctx)
+        public static void ThrowActivity([ActivityTrigger] IDurableActivityContext ctx)
         {
             string message = ctx.GetInput<string>();
             throw new Exception(message);
         }
 
+        public static Task<string> ReadStringFromTextFile([ActivityTrigger] IDurableActivityContext ctx)
```

Comment:
It looks like there are a few activity functions and actor functions which aren't being used yet. I assume that's intentional, but wanted to call it out just in case.

Reply:
Yes, this is actually the one which I was working on last week. I have a test / example implemented in a different branch but I am not sure I should add it to this PR.